tests/test_daemon.cpp

```diff
@@ -164,6 +166,16 @@ a few more tests for `false`, since there are different portions of code dependi
     mpt::MockJsonUtils& mock_json_utils = *mock_json_utils_injection.first;
 };
 
+TEST_F(Daemon, buildsConfig)
+{
+    // Clear our stub image host so that DaemonConfigBuilder::build generates the default.
+    config_builder.image_hosts.clear();
+
+    auto config = config_builder.build();
+    EXPECT_EQ(config->ssh_username, "ubuntu");
+    EXPECT_THAT(config->image_hosts, Not(IsEmpty()));
+}
+
```

Comment:
This test is pretty simple and is mainly just to ensure that we can auto-populate `image_hosts`. Maybe we could add a few more `EXPECT_*` calls in here...